src/js/components/Chart.vue

```diff
@@ -1,11 +1,20 @@
 <template>
   <div class="chart-container">
     <span class="revenue-label">Total <strong>Revenue</strong></span>
-    <svg class="revenue-line" width="100%" height="310px">
-      <line :y1="Y0" :y2="Y0 + HEIGHT * scale" x1="0" x2="0" />
-    </svg>
-    <svg id="chart" data-chart width="100%" height="310px">
+    <svg
+      id="chart"
+      data-chart
+      xmlns="http://www.w3.org/2000/svg"
+      viewBox="0 0 700 325"
```

Comment:
Why `700` here? It looks like this still cuts off some charts, e.g. the "LULU 20221" example.

Reply:
@jgerigmeyer I used some of the variables for calculating the widths of the chart pieces for this, since the viewBox wants an aspect ratio, and each chart is different based on what it contains. LMK if this works better for our purposes.